src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -427,6 +427,52 @@ export class PostgresCohortMembersService {
         );
       }
 
+      // NEW: Add cohort details to each user
+      if (results['userDetails'] && results['userDetails'].length > 0) {
```

ESLint

[error] Replace `'userDetails']·&&·results['userDetails'` with `"userDetails"]·&&·results["userDetails"`

(prettier/prettier)